boot.php

```diff
@@ -15,7 +23,12 @@
     throw new RuntimeException('You need to pass the ROM file name (Ex: drmario.rom)');
 }
 
-$filename = 'roms/'.$argv[1];
+if(file_exists(getcwd().'/'.$argv[1])){
+    $filename = getcwd().'/'.$argv[1];
+} else{
+    $filename = $argv[1];
```

Comment:
You should just let this line and remove the entire `if/else` statement.

`getcwd().'/'.` is already the current working directory, the PHP script can read absolute and relative paths.